enterprise/setup/index.mdx

```diff
@@ -282,6 +282,10 @@ AWS_REGION=us-east-1
 
 ### Appendix D - LDAP configuration
 
+If configuring LDAP after initially setting up an organization with standard password-based login,
+be sure to set DEFAULT_ORG_ROLE (see above) to ensure that new LDAP users are added to the existing
+organization.
```

Comment:
I think we can actually state this more generally, as you'd run into issues here regardless of how the organization is set up:

```
When using LDAP, it is recommended to also set DEFAULT_ORG_ROLE to a non-standard value (see above) to ensure that new LDAP users are added to the existing organization.
```